src/engraving/style/style.h

```diff
@@ -100,6 +105,8 @@ class MStyle
 
     void readVersion(String versionTag);
     int m_version = 0;
+    ScoreStylePreset m_preset = ScoreStylePreset::DEFAULT;
```

Comment:
Since the type name is mentioned elsewhere on the line, you can also do:

```C++
auto m_preset = ScoreStylePreset::DEFAULT;
```

But writing it explicitly is fine too.